source/common/http/conn_manager_impl.h

```diff
@@ -351,6 +351,8 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
       return *tracing_custom_tags_;
     }
 
+    bool validateHeaders();
```

Comment:
comment, especially as today it doesn't always validate

Reply:
Done